app/views/account/tenant/_form.html.slim

```diff
@@ -0,0 +1,6 @@
+    = simple_form_for tenant, url: form_path,
```

Comment:
there's no need to override url attribute for resourceful objects,
simple form knows which action to perform for create/update record if you pass object instance

Reply:
I deleted this file. Please see if this one is correct - https://github.com/Ruby-IF081/heroes_team_app/pull/83/files#diff-5d538d60f486696d6b27bfb0108b350a